macOS/DuckDuckGo/Fire/ViewModel/FireDialogViewModel.swift

```diff
@@ -176,7 +176,10 @@ final class FireDialogViewModel: ObservableObject {
     private(set) var shouldShowPinnedTabsInfo: Bool = false
 
     var shouldShowChatHistoryToggle: Bool {
-        aiChatHistoryCleaner.shouldDisplayCleanAIChatHistoryOption && mode.shouldShowChatHistoryToggle && clearingOption.shouldShowChatHistoryToggle
+        let isPresentedOnAIChatTab = tabCollectionViewModel?.selectedTab?.url?.isDuckAIURL ?? false
+        return aiChatHistoryCleaner.shouldDisplayCleanAIChatHistoryOption
+        && mode.shouldShowChatHistoryToggle
+        && (clearingOption.shouldShowChatHistoryToggle || isPresentedOnAIChatTab)
```

Comment:
Nitpicky - but I'd consider indenting this a bit as it looks a bit odd right now.